src/libMessage/Messenger.cpp

```diff
@@ -5432,6 +5518,45 @@ bool Messenger::GetLookupSetStateDeltaFromSeed(const bytes& src,
   return true;
 }
 
+bool Messenger::GetLookupSetStateDeltasFromSeed(
+    const bytes& src, const unsigned int offset, uint64_t& lowBlockNum,
+    uint64_t& highBlockNum, PubKey& lookupPubKey, vector<bytes>& stateDeltas) {
+  LOG_MARKER();
+
+  LookupSetStateDeltasFromSeed result;
+
+  result.ParseFromArray(src.data() + offset, src.size() - offset);
+
+  if (!result.IsInitialized()) {
+    LOG_GENERAL(WARNING, "LookupSetStateDeltasFromSeed initialization failed");
+    return false;
+  }
+
+  lowBlockNum = result.data().lowblocknum();
+  highBlockNum = result.data().highblocknum();
```

Comment:
It will be better to have a check on the size of delta and the (highBlockNum - lowBlockNum) is needed

Reply:
Fixed